main.py

```diff
@@ -1,4 +1,280 @@
-import glob, os
-os.chdir('./res')
-for file in glob.glob("*.fb2"):
-    print(file)
+import glob
+import os
+import shutil
+import xml.etree.ElementTree as ET
+from loguru import logger
+import collections
+import re
+import sqlite3
+from sqlite3 import Error
+import timeit
+
+
+# Class provides functionality to search for fb2 files
+class Fb2Reader:
+
+    @logger.catch()
+    def fb2_file_search(self):
+        os.chdir('./res/input')
+        self.__incorrect_file_removal()
+        fb2_file_list = {file for file in glob.glob("*.fb2")}
+        logger.info('File search is done')
+        return fb2_file_list
+
+    @logger.catch()
+    def parse_fb2(self, fb2_file):
+        try:
+            tree = ET.parse(fb2_file)
+            logger.info('File is parsed')
+        except Error as e:
+            logger.error('File is not parsed')
+        return tree
+
+    @logger.catch()
+    def __incorrect_file_removal(self):  # private
+        file_list = {file for file in glob.glob("*.*") if file not in glob.glob("*.fb2")}
```

Comment:
Логика реализована правильно. Можно немного улучшить перфоманс, не перебирая файлы в директории 2-й раз:
```suggestion
        file_list = {file for file in glob.glob("*.*") if os.path.splitext(file)[1] != '.fb2'}
```